src/style.css

```
@@ -0,0 +1,115 @@
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir.ttc");
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  box-sizing: border-box;
+  color: #3e4e61;
+  font-weight: 300;
+  font-style: normal;
+}
+
+.container-input {
+  margin-bottom: 20px;
+  margin-top: 20px;
+}
+
+.img-input-glass {
+  left: 34px;
+  top: 45px;
+  width: 19px;
+  height: 19px;
+  background-image: url(/images/Search.svg);
+  position: absolute;
+  background-repeat: no-repeat;
+}
+
+.input-biggest {
+  width: 100%;
+  height: 70px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  background-repeat: no-repeat;
+  background-position: 25px 24px;
+  padding-left: 62px;
+  outline: none;
+  font-size: 16px;
+  font-family: Avenir-Heavy, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+}
+
+.container-input > .input-biggest::placeholder {
```

First class name is redundant here
```suggestion
.input-biggest::placeholder {
```